src/board-explorer-app/board-explorer-app.html

```diff
@@ -470,12 +470,12 @@
             </div>
             <div hidden$="[[isEmpty(active.connected)]]">
               Connected to: <template is="dom-repeat" items="[[active.connected]]" as="refdes">
-                <a href="#[[board]]/[[refdes]]">[[lookupName(refdes)]]</a>
+                <a on-mouseover="onAnchorHoverIn" on-mouseout="onAnchorHoverOut" href="#[[board]]/[[refdes]]">[[lookupName(refdes)]]</a>
               </template>
             </div>
             <div hidden$="[[isEmpty(active.related)]]">
               Related to: <template is="dom-repeat" items="[[active.related]]" as="refdes">
-                <a href="#[[board]]/[[refdes]]">[[lookupName(refdes)]]</a>
+                <a on-mouseover="onAnchorHoverIn" on-mouseout="onAnchorHoverOut" href="#[[board]]/[[refdes]]">[[lookupName(refdes)]]</a>
               </template>
```

Comment:
The above could be replaced by adding a `this.querySelectorAll('a')` in this code chunk in board-explorer-app.html:

```javascript
        /* If this is the item currently being hovered on, notify the UI
         * to update the DOM */
        if (this.$.boardViewer.selected.length &&
            this.$.boardViewer.selected[0] == item.refdes) {
          Polymer.dom(this.$.documentation).innerHTML = item.html;
/* add  a Polymer.flush(); and then the querySelector, then forEach() on the results to add the event listeners for anchor in, anchor out */
        }
```